core/msd/src/main/rdl/Workload.rdli

```diff
@@ -145,4 +145,36 @@ resource BulkWorkloadResponse POST "/workloads" (name=getWorkloadsByDomainAndSer
         ResourceError UNAUTHORIZED;
         ResourceError TOO_MANY_REQUESTS;
     }
-}
\ No newline at end of file
+}
+
+// Api to discover an additional instance which can have static or dynamic or both IPs
+resource Workloads PUT "/domain/{domainName}/service/{serviceName}/workload/discover/instance" (name=postCompositeInstance) {
```

Comment:
since this is a PUT API, the name should not refer to it as postCompositeInstance and instead should be called putCompositeInstance